doc/changelog.rst

```diff
@@ -41,13 +43,17 @@ Detailed Notes
   there is now an available ONNX wheel for use with Python 3.10, and wheels for
   all of SmartSim's machine learning backends with Python 3.11.
   (SmartSim-PR451_) (SmartSim-PR461_)
+- SmartSim can now be built and used on platforms using Apple Silicon
+  (ARM64). Currently, only the pyTorch backend is supported. Note that libtorch
```

Comment:
Correct me if I'm wrong but isn't the preferred capitalization
```suggestion
  (ARM64). Currently, only the PyTorch backend is supported. Note that libtorch
```
? At least I thought that was how they titled their releases.

Reply:
You're totally correct

Reply:
Re-surfacing this `pyTorch` -> `PyTorch`